src/Microsoft.AspNet.Server.Kestrel/Filter/SocketInputStream.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using IllyriadGames.ByteArrayExtensions;
```

Comment:
Added reference as specific version number; so the code referenced is under nuget's control not mine :wink: